x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/types.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum OpenModal {
+  None = 'None',
+  ConfirmModifyModal = 'Confirm Modify',
```

Comment:
FWIW, I can't seem to get a confirm modify modal to pop up on the standalone UI 🤔 I only see the confirmation modal after clicking the restore defaults / save button. Any tips?

Reply:
Separately, it's interesting to me that we use a modal for this page only whereas the other "Restore defaults" pages mostly just use a basic `window.confirm`. Wondering if we should run that by @daveyholler at some point

Reply:
Good call. I made a note of this, let's revisit when we add the UI.

And the confirm modify it triggered here: https://github.com/elastic/kibana/pull/94629/files#diff-91d6dae72a4d40fdbc4f847e58264f6fbd97ed01af1107082469e9a56f0f0507R111-R112

It looks like when we initialize data and all of the data matches the defaults 🤔 . I'll revisit this when I add the UI.

Reply:
Sounds good on deferring, will try to remember to bring this back up!

Reply:
Lol so I just checked the standalone UI and it's this "modal"/popover??? which I legit hadn't even guessed was a modal or at least the same modal as the other confirmations

<img width="663" alt="" src="https://user-images.githubusercontent.com/549407/111349449-5d84a300-863e-11eb-8a71-81a76c6f3d32.png">

I find that modal really weird personally / honestly and I don't like the extra click it takes to actually see the UI - let me see if Davey has an easy option for us to remove (and not add too much extra code) here

Reply:
It's easy enough to remove I believe. I'll remove it as part of PR feedback.

Reply:
3fd186469b6f5bf7e9be7f85394e14cce184833d